usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1707,7 +1684,7 @@ static CK_RV cca_get_adapter_domain_selection_infos(STDLL_TokData_t *tokdata)
      */
      cca_private->dev_any = FALSE;
      cca_private->dom_any = FALSE;
-#endif
+#else
```

Comment:
You don't need to set `dev_any` and `dom_any` to FLASE. They are already FALSE by default (and will stay FALSE if you ifdef out the getenvs).
So I would remove the `#if defined(_AIX)` block and make it a `#if !defined(_AIX)` instead of the `#else` block, but keep the comment about AIX.